src/main/java/wooteco/subway/path/infrastructure/ShortestPathWithDijkstra.java

```diff
@@ -0,0 +1,30 @@
+package wooteco.subway.path.infrastructure;
+
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+import org.springframework.stereotype.Component;
+import wooteco.subway.line.domain.Section;
+import wooteco.subway.path.domain.Path;
+import wooteco.subway.station.domain.Station;
+
+import java.util.List;
+
+@Component
+public class ShortestPathWithDijkstra implements ShortestPath {
+    @Override
+    public Path getPath(Station source, Station target, List<Section> sections) {
+        WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph(DefaultWeightedEdge.class);
```

Comment:
아래 부분에도 인텔리제이에서 노란불이 뜨는데 확인 부탁드려요 :) 
```suggestion
        WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph<>(DefaultWeightedEdge.class);
```

Reply:
이곳에 경고가 엄청 많았네요 ㄷㄷ
현재는 아래와 같이 수정됐어요!

```
WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph<>(DefaultWeightedEdge.class);
```